compose/release-notes.md

```diff
@@ -7,6 +7,69 @@ redirect_from:
   - /release-notes/docker-compose/
 ---
 
+## 1.28.0 
+(2021-01-20)
+
+### Features
+
+- Added support for Nvidia GPUs via device requests.
+
+- Added support for service profiles.
+
+- Changed the SSH connection approach to the Docker CLI's via shellout to the local SSH client (old behaviour enabled by setting `COMPOSE_PARAMIKO_SSH` environment variable).
+
+- Added flag to disable log prefix.
+
+- Added flag for ansi output control.
+
+### Bugs
+
+- Made `parallel_pull=True` by default.
+
+- Restored the warning for configs in non-swarm mode.
+
+- Took `--file` into account when defining `project_dir`.
+
+- Fixed service attach bug on `compose up`.
+
+### Miscellaneous
+
+- Made COMPOSE_DOCKER_CLI_BUILD=1 the default.
```

Comment:
```suggestion
- Made `COMPOSE_DOCKER_CLI_BUILD=1` the default.
```

Reply:
Perhaps move this to "features" (not sure). I moved https://github.com/docker/docker.github.io/pull/11858 out of "draft" (which is related to this change).

```suggestion
- Docker Compose now uses the native Docker CLI's `build` command when building images. Set the `COMPOSE_DOCKER_CLI_BUILD=0` environment variable to disable this feature.
```